src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -160,6 +160,28 @@ public function loadScripts() {
 		";
 		wp_add_inline_style( 'give-sequoia-template-css', $feeRecoveryDynamicCss );
 
+		$mailchimpDynamicCss = "
+			.give-mailchimp-fieldset:hover,
+			.give-mailchimp-fieldset.active {
+				border: 1px solid {$primaryColor} !important;
+			}
+			.give-mailchimp-fieldset input[type='checkbox'] + span::after {
+				background-image: url(\"data:image/svg+xml,%3Csvg width='15' height='11' viewBox='0 0 15 11' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M5.73047 10.7812C6.00391 11.0547 6.46875 11.0547 6.74219 10.7812L14.7812 2.74219C15.0547 2.46875 15.0547 2.00391 14.7812 1.73047L13.7969 0.746094C13.5234 0.472656 13.0859 0.472656 12.8125 0.746094L6.25 7.30859L3.16016 4.24609C2.88672 3.97266 2.44922 3.97266 2.17578 4.24609L1.19141 5.23047C0.917969 5.50391 0.917969 5.96875 1.19141 6.24219L5.73047 10.7812Z' fill='%23{$rawColor}'/%3E%3C/svg%3E%0A\") !important;
+			}
+		";
+		wp_add_inline_style( 'give-sequoia-template-css', $mailchimpDynamicCss );
```

Comment:
Is this added regardless of whether the add-on is active or not?

Reply:
For now, yes. The dynamic css is grouped by addon so that as we introduce functionality to reliably check whether an addon is enabled (@ravinderk has been working on this as part of the receipt compatibility) we will be able to load addon specific CSS only when appropriate.

Reply:
@DevinWalker we have to add conditions to check if the addon is active or not to limit dynamic CSS size.
@henryholtgeerts can you explain what you are expecting?

@JasonTheAdams Can you add your thoughts on this because I think adding conditions to output only addon related dynamic style will be overdoing.

Reply:
I agree that we should limit assets to loading only when needed, otherwise this is a slippery slope. Perhaps at some point we'll come up with a more formal system for checking for "features" versus addons, but in the meantime is there any reason we couldn't just use [is_plugin_active](https://developer.wordpress.org/reference/functions/is_plugin_active/) here?

Reply:
@JasonTheAdams `class_exists` will be quicker than `is_plugin_active` and this function also not available on the frontend, so we have to load the extra file.

can we use `class_exists` on `defined`?

Reply:
I'm reluctant to use `class_exists` as we're not _actually_ using any kind of addon class, here. And we'd be coupling GiveWP core to the shape of the addon, ambiguously. The `is_plugin_active` function is fairly performant as the `active_plugins` option will certainly be autoloaded and cached, so it's just checking in an array.

To make it available we'd just need to do
`include_once ABSPATH . 'wp-admin/includes/plugin.php';`

Reply:
@JasonTheAdams, In this case, I will suggest creating a wrapper function around it.

```
namespace Give\Helpers\Utils

class Utils{
     publish static function isPluginActive( $plugin ){
         if( ! function_exists( 'is_plugin_active')  ){
             include_once ABSPATH . 'wp-admin/includes/plugin.php';
         }

         return is_plugin_active( $plugin );
     }
}
```

Let me know what do you think.

Reply:
Yeah, I'm cool with that helper. Go for it.

Reply:
@ravinderk Are you implementing this in this PR?

Reply:
@JasonTheAdams one more thought on this because we have to hardcode addon file path in `isPluginActive`, I want to suggest one more abstraction. Can we create a helper class for each addon?

```
namespace Give\Helpers\Addon\Recurring;

class Recurring{
    private static $plugin = 'give-recurring/give-recurring.php';

    public static function isPluginActive(){
        return Utils::isPluginActive( self::plugin )
    }
}
```

so instead of using `Utils::isPluginActive( 'give-recurring/give-recurring.php' )` we can use `RecurringAddon::isPluginActive()`.

Let me know what do you think.

Reply:
@ravinderk I like where your head is at, I just don't have as much context. Do those addon classes already exist? Do they do something other than check if the addon is active?

Also, slight tweak:
```php
<?php
namespace Give\Helpers\Addon\Recurring;

class Recurring{
    const PLUGIN_FILE = 'give-recurring/give-recurring.php';

    public static function isActive(){
        return Utils::isPluginActive( self::PLUGIN_FILE );
    }
}
```

That said, I don't want to abstract the crap out of this right now. If creating a single helper keeps this PR moving forward, I'd prefer that. We can optimize more, later.

Reply:
I'm gonna move forward with implementing the Helper method in this PR, and suggest we have a more robust conversation about these more abstracted classes on a GiveWP dev call/in a separate issue. I think it will ultimately be the way to go but I'm hesitant to add a bunch of new classes as part of this PR, especially if they all serve the same basic utility starting out.

Reply:
@henryholtgeerts I implemented function in same pr: https://github.com/impress-org/givewp/compare/4df865e3a942...d970a0e0981f

Reply:
The helper method has been implemented in the PR, ref: https://github.com/impress-org/givewp/pull/4771/commits/e6c90b439cfd88c9375308c8c33256abaef121e6